catalog/dags/providers/provider_api_scripts/science_museum.py

```diff
@@ -154,22 +154,18 @@ def get_record_data(self, record):
 
     @staticmethod
     def _get_creator_info(attributes):
-        creator_info = None
-        if (life_cycle := attributes.get("lifecycle")) is not None:
-            creation = life_cycle.get("creation")
-            if isinstance(creation, list):
-                maker = creation[0].get("maker")
-                if isinstance(maker, list):
-                    creator_info = maker[0].get("summary_title")
-        return creator_info
+        if not (maker := attributes.get("creation", {}).get("maker", [])):
+            return None
+
+        return maker[0].get("summary", {}).get("title", None)
```

Comment:
Hm, I can't decide -- theoretically there could be a difference between "no maker information was provided by the source"  and "an authoritative (museum) source confirmed the maker is unknown". But maybe that's not a useful distinction. I would be curious if we do something similar for any of our other sources 🤔 

I'll make a separate issue for this, mostly because it looks like we do have "Unknown maker" in production data at the moment.

Reply:
#4145 created!